rules-generated/quarkus/jsonp/jsonp-to-quarkus.windup.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="jsonp-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate jsonp components to Quarkus extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="jsonp" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="quarkus-jsonp-00000">
+            <when>
+                <project>
+                    <artifact groupId="org.glassfish" artifactId="jakarta.json"/>
+                </project>
+            </when>
+            <perform>
+                <hint title="Replace the 'jakarta.json' dependency with Quarkus 'quarkus-jsonp' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `org.glassfish:jakarta.json` dependency with the Quarkus dependency `io.quarkus:quarkus-jsonp`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/" />
```

Comment:
No guide nor quickstart found.
I would keep the generic link to the Quarkus guides page